### ISSUE TITLE
A11y_Azure VM Image Builder_Azure Image Builder Preview_ScreenReader: NVDA is reading the label of 'Enabled' checkbox twice

### ISSUE THREAD

**Issue by saiprasanna93** [nvaccess/nvda#10139]
Environment Details:
URL: https://acctesting.visualstudio.com​
Browser Details: Google Chrome Version 74.0.3729.169(Official Build) (64-bit)​
​
Repro Steps:​​
1)Hit URL: https://acctesting.visualstudio.com and log in with appropriate credentials.​​
2)Open any project from the available projects​
3)Tab till Pipelines and press enter​
4)Tab till Releases and press enter​
5)Create a new release pipeline​
6)Select Empty Job and tab till Add an Artifact and press enter​
7)Select Azure Repos Git and fill the required fields and tab till 'Add' button and press enter​
8)Tab till stages and select job​
9)select 'Agent job' and add task​
10)Search for Image Builder Task and Press enter on Add button​
11)Select Azure Image Builder Task under Agent Job​
12)Tab till 'Enabled' check box​
12)Verify how NVDA is reading the label of enabled checkbox ​
​
Actual Result:​
NVDA is reading the label of enabled checkbox twice​
​
Expected result: ​
NVDA should read the label of enabled checkbox only once​
​
User Impact: ​
Screen reader users will face problem if NVDA is reading any label of a control twice
​
MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by saiprasanna93** [comment]
![Bug_10139_NVDA is reading the label of 'Enabled' checkbox twice](https://user-images.githubusercontent.com/50437422/63575752-7636ef00-c5a8-11e9-81c5-8b45e54de694.png)

**Comment by Adriani90** [comment]
@feerrenrut is this a duplicate of #9155?

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
